infra/reachability-metadata/src/main/resources/META-INF/native-image/com.github.ben-manes.caffeine/caffeine/2.9.3/reflect-config.json

```diff
@@ -18,5 +18,103 @@
+  "name":"com.github.benmanes.caffeine.cache.BaseMpscLinkedArrayQueueColdProducerFields",
+  "fields":[{"name":"producerLimit"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.BaseMpscLinkedArrayQueue"},
+  "name":"com.github.benmanes.caffeine.cache.BaseMpscLinkedArrayQueueConsumerFields",
+  "fields":[{"name":"consumerIndex"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.BaseMpscLinkedArrayQueue"},
+  "name":"com.github.benmanes.caffeine.cache.BaseMpscLinkedArrayQueueProducerFields",
+  "fields":[{"name":"producerIndex"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.LocalLoadingCache"},
+  "name":"com.github.benmanes.caffeine.cache.CacheLoader",
+  "methods":[{"name":"loadAll","parameterTypes":["java.lang.Iterable"] }]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.PD"},
+  "name":"com.github.benmanes.caffeine.cache.PD",
+  "fields":[{"name":"value"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.PS"},
+  "name":"com.github.benmanes.caffeine.cache.PS",
+  "fields":[{"name":"key"}, {"name":"value"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.PSA"},
+  "name":"com.github.benmanes.caffeine.cache.PSA",
+  "fields":[{"name":"accessTime"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.SSMSA"},
+  "name":"com.github.benmanes.caffeine.cache.PSAMS",
+  "methods":[{"name":"<init>","parameterTypes":[] }]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.PSW"},
+  "name":"com.github.benmanes.caffeine.cache.PSW",
+  "fields":[{"name":"writeTime"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.SSMSW"},
+  "name":"com.github.benmanes.caffeine.cache.PSWMS",
+  "methods":[{"name":"<init>","parameterTypes":[] }]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.BoundedLocalCache$BoundedLocalManualCache"},
+  "name":"com.github.benmanes.caffeine.cache.SSMSA",
+  "methods":[{"name":"<init>","parameterTypes":["com.github.benmanes.caffeine.cache.Caffeine","com.github.benmanes.caffeine.cache.CacheLoader","boolean"] }]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.BoundedLocalCache$BoundedLocalManualCache"},
+  "name":"com.github.benmanes.caffeine.cache.SSMSW",
+  "methods":[{"name":"<init>","parameterTypes":["com.github.benmanes.caffeine.cache.Caffeine","com.github.benmanes.caffeine.cache.CacheLoader","boolean"] }]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.StripedBuffer"},
+  "name":"com.github.benmanes.caffeine.cache.StripedBuffer",
+  "fields":[{"name":"tableBusy"}]
+},
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.StripedBuffer"},
+  "name":"java.lang.Thread",
+  "fields":[{"name":"threadLocalRandomProbe"}]
+},
```

Comment:
- I'm curious if there are any known failing unit tests. Because this JSON has been submitted by me to OracleLabs side, see https://github.com/oracle/graalvm-reachability-metadata/blob/0.3.8/metadata/com.github.ben-manes.caffeine/caffeine/2.9.3/reflect-config.json#L175 .
- `java.lang.Thread` and `sun.misc.Unsafe` usually have lower-level callers, and those low-level callers may not be Caffeine.